execution/executor-test-helpers/src/integration_test_impl.rs

```diff
@@ -161,76 +167,27 @@ pub fn test_execution_with_storage_impl() -> Arc<AptosDB> {
             vec![],
         )),
     );
-
+    
     // Transfer 20k coins from account1 to account2.
     // balance: <1.98M, 1.22M, 1M
-    let txn4 = get_test_signed_transaction(
-        account1,
-        /* sequence_number = */ 0,
-        privkey1.clone(),
-        pubkey1.clone(),
-        Some(encode_peer_to_peer_with_metadata_script_function(
-            xus_tag(),
-            account2,
-            20_000,
-            vec![],
-            vec![],
-        )),
-    );
-
+    let txn4 = local_account1.sign_with_transaction_builder( txn_factory.transfer(account2, 20_000));
     // Transfer 10k coins from account2 to account3.
     // balance: <1.98M, <1.21M, 1.01M
-    let txn5 = get_test_signed_transaction(
-        account2,
-        /* sequence_number = */ 0,
-        privkey2,
-        pubkey2,
-        Some(encode_peer_to_peer_with_metadata_script_function(
-            xus_tag(),
-            account3,
-            10_000,
-            vec![],
-            vec![],
-        )),
-    );
+    let txn5 = local_account2.sign_with_transaction_builder( txn_factory.transfer(account3, 10_000));
 
     // Transfer 70k coins from account1 to account3.
     // balance: <1.91M, <1.21M, 1.08M
-    let txn6 = get_test_signed_transaction(
-        account1,
-        /* sequence_number = */ 1,
-        privkey1.clone(),
-        pubkey1.clone(),
-        Some(encode_peer_to_peer_with_metadata_script_function(
-            xus_tag(),
-            account3,
-            70_000,
-            vec![],
-            vec![],
-        )),
-    );
+    let txn6 = local_account1.sign_with_transaction_builder( txn_factory.transfer(account3, 70_000));
 
-    let block1 = vec![tx1, tx2, tx3, txn1, txn2, txn3, txn4, txn5, txn6];
+    let block1 = vec![tx1, tx2, tx3, txn1, txn2, txn3, UserTransaction(txn4), UserTransaction(txn5), UserTransaction(txn6)];
```

Comment:
we should replace all tx1, tx2 etc too

Reply:
So I think I did it for txn 1, 2 and 3 but I am not sure how to do it for tx1, 2 and 3...

Reply:
I don't see txn 1, 2, 3 updated, we can convert tx 1, 2, 3 in another PR

Reply:
It's on my latest commit [Better engineering] fixed txn1 2 and 3